desk/app/chat.hoon

```diff
@@ -4,13 +4,14 @@
 /-  e=epic
 /-  contacts
 /+  default-agent, verb-lib=verb, dbug
-/+  chat-json
 /+  pac=dm
 /+  ch=chat-hark
 /+  gra=graph-store
 /+  epos-lib=saga
 /+  wood-lib=wood
 /+  mig=chat-graph
+::  performance, keep warm
+/+  chat-json
```

Comment:
Is this (and all the other `*-json` lib imports in the %groups desk) still necessary with `lib/mark-warmer` being imported by `:groups-ui`?

Reply:
good question I'm not sure